raw-cms-app/src/modules/core/views/entity-details-view/entity-details-view.js

```diff
@@ -31,6 +32,8 @@ const _EntityDetailsView = async (res, rej) => {
         this.title = isNew
```

Comment:
No need for `title` property on component, see comment on "raw-cms-app/src/modules/core/views/collection-table-view/collection-table-view.js".

Reply:
I have removed title var from data, which is no longar in use. where as entity-details-view.js, title is being updated by updateTitle() method on eventbus, i just following that. rather then dispatching title from mounted property.